### ISSUE TITLE
displaylink download version 1.9.0 instead of 1.1.0

### ISSUE THREAD

**Issue by it-vsg** [Installomator/Installomator#1427]
Since about few days the displaylinkmanager label installing the old version 1.9.0 instead of version 1.1.0.
Before it was correct.

**Comment by asri-tm** [comment]
The issue here is synaptics keep moving the latest PKG download around on https://www.synaptics.com/products/displaylink-graphics/downloads/macos

replacing downloadURL with the following should fix it for now until Synaptics decides to move it down again.

```
downloadURL=https://www.synaptics.com$(redirect=$(curl -sfL https://www.synaptics.com/products/displaylink-graphics/downloads/macos | grep 'class="download-link">Download' | sed -n '1p' | sed 's/.*href="//' | sed 's/".*//') && curl -sfL "https://www.synaptics.com$redirect" | grep 'class="no-link"' | awk -F 'href="' '{print $2}' | awk -F '"' '{print $1}')
```

**Comment by lfittarelli** [comment]
We're seeing the same issue currently on macOS devices where DisplayLInk Manager 1.10 is installed, and it "updates" to 1.9.

**Comment by TheRossRoss** [comment]
Not sure if this helps but this is the URL we managed to get it to install from but were not sure how to adjust the label to use the new URL

https://www.synaptics.com/products/displaylink-graphics/downloads/macos-connectivity-1.10.2?filetype=exe

**Comment by DanielRossEUPE** [comment]
How can I help change this with the URL above so the label can use it?

**Comment by acodega** [comment]
Believe this is fixed in Installomator 10.6
